packages/components/skeleton/src/SkeletonContainer/SkeletonContainer.tsx

```diff
@@ -48,108 +48,104 @@ export interface SkeletonContainerProps extends CommonProps {
   children: React.ReactNode;
 }
 
-export const SkeletonContainer = forwardRef<SVGElement, SkeletonContainerProps>(
-  ({
-    children,
-    testId = 'cf-ui-skeleton-form',
-    ariaLabel = 'Loading component...',
-    width = '100%',
-    height = '100%',
-    preserveAspectRatio,
-    backgroundColor = '#e5ebed',
-    backgroundOpacity = 1,
-    isAnimated = true,
-    speed = 2,
-    foregroundColor = '#f7f9fa',
-    foregroundOpacity = 1,
-    svgWidth = '100%',
-    svgHeight = '100%',
-    clipId,
-    gradientId,
-    animateId,
-    ...otherProps
-  }) => {
-    const uniqueClipId = useId(clipId, 'cf-ui-skeleton-clip');
-    const uniqueGradientId = useId(gradientId, 'cf-ui-skeleton-clip-gradient');
-    const randomAnimateId = useId(undefined, 'animation');
-    const uniqueAnimateId = animateId || randomAnimateId;
+export function SkeletonContainer({
```

Comment:
Shouldn't this be in another PR? Or is it relevant to Modal changes?

Reply:
I had to fix it in order to procceed.